File: Algorytm_Warunkowy/alg_warunkowy1.cpp
/*
 * alg_warunkowy1.cpp
 */


#include <iostream>
using namespace std;

int wynik = 0;


int main(int argc, char **argv)
{
	cout << "Podaj wynik" << endl;
	cin >> wynik;
	cout << "Twój wynik to: " << wynik << endl;
	
	if (wynik < 0 || wynik > 60){
		cout << " Błędne dane!";
	}
	else if (wynik < 20){
		cout << " Grupa podstawowa!";
	}
	else if (wynik <= 40){
		cout << " Grupa średniozaawansowana!";
	}
	else cout << " Grupa zaawansowana!";
	return 0;
}

File: Algorytm_Warunkowy/alg_warunkowy1.py
#
#  alg_warunkowy.py
# 


def main(args):
    
    wynik = int(input("Podaj wynik! "))
    print("Twój wynik to: ", wynik)
    
    if wynik < 0 or wynik > 60:
        print("Błędne dane!")
    
    elif wynik < 20:
        print("Grupa podstawowa!")
    
    elif wynik <=40:
        print("Grupa średniozaawansowana!")
    
    else:
        print("Grupa zaawansowana!")
        
    return 0

if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))
